Microsoft.NET.Build.Containers/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -1,4 +1,18 @@
 <Project>
+    <Target Name="_ContainerVerifySDKVersion"
+        Condition="'$(WebPublishMethod)' == 'Container' or '$(PublishProfile)' == 'DefaultContainer'"
+        BeforeTargets="AfterPublish">
```

Comment:
we can't rely on the Container target hookups having been executed in a build, so we need to establish some reasonable point at which to do this check. AfterPublish seemed good.